stingray/crossspectrum.py

```diff
@@ -1406,6 +1406,76 @@ def from_lightcurve(
             gti=gti,
         )
 
+    @staticmethod
```

Comment:
The only problem with that is that the time series object does not have a default flux attribute, like lightcurve (which has `counts`). Using this method, you can specify the flux attribute. We could in principle add the flux keyword to `__init__`, but I'm not sure it's very helpful

Reply:
Let's not worry about it for this version. I'll point out that one *can* make power spectra of all sorts of time series that aren't light curves, but I don't think it's worth worrying about it for now. It would be good to showcase this in the Powerspectrum tutorial to make sure users know to use a different syntax for TimeSeries than for LightCurve